dom5.ts

```diff
@@ -12,13 +12,10 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-/// <reference path="./custom_typings/main.d.ts" />
 import * as cloneObject from 'clone';
-import * as parse5 from 'parse5';
-import {ASTNode as Node} from 'parse5';
-export {ASTNode as Node} from 'parse5';
```

Comment:
this'll be visible downstream, breaking tsc runs. pretty much only we are using it, and the fix is simple enough, but we'll need to track down the broken projects and update them